seqr/views/utils/terra_api_utils_tests.py

```diff
@@ -148,3 +159,10 @@ def test_user_get_workspace_access_level(self, mock_time, mock_logger):
         mock_logger.warning.assert_called_with(
             'test_user called Terra API: GET /api/workspaces/my-seqr-billing/my-seqr-workspace?fields=accessLevel got status 404 with reason: Not Found')
         self.assertEqual(len(mock_logger.method_calls), 1)
+
+        # Test cache hit
+        url = '{}api/workspaces/my-seqr-billing/my-seqr-workspace?fields=accessLevel'.format(TEST_TERRA_API_ROOT_URL)
+        mock_redis.return_value.get.return_value = '{"accessLevel": "READER"}'
+        permission = user_get_workspace_access_level(user, 'my-seqr-billing', 'my-seqr-workspace')
+        self.assertDictEqual(permission, {"accessLevel": "READER"})
+        mock_redis.return_value.get.assert_called_with('terra_req__ec4582d9e5e0247385d63538686095cc')
```

Comment:
can you add checks here that no requests were made

Reply:
okay